gamemodes/terrortown/gamemode/server/sv_eventpopup.lua

```diff
@@ -0,0 +1,48 @@
+util.AddNetworkString("ttt2_eventpopup")
+
+EPOP = EPOP or {}
+
+---
+-- Adds a translated popup message to the @{EPOP}
+-- @param table|Player plys A table of player that should receive this popup, broadcasts it if nil
+-- @param string|table title The title of the popup that will be displayed in large letters (can be a table with `text` and `color` attribute)
+-- @param [opt]string|table subtitle An optional description that will be displayed below the title (can be a table with `text` and `color` attribute)
+-- @param [default=4]number displayTime The render duration of the popup
+-- @param [default=true]boolean blocking If this is false, this message gets instantly replaced if a new message is added
+-- @realm server
+function EPOP:AddMessage(plys, title, subtitle, displayTime, blocking)
+	net.Start("ttt2_eventpopup")
+
+	if title or isstring(title) then
+		net.WriteBool(false)
+		net.WriteString(title or "")
+	elseif not IsValid(title.color) then
+		net.WriteBool(false)
+		net.WriteString(title.text or "")
+	else
+		net.WriteBool(true)
+		net.WriteString(title.text or "")
+		net.WriteColor(title.color or COLOR_WHITE)
+	end
+
+	if not subtitle or isstring(subtitle) then
+		net.WriteBool(false)
+		net.WriteString(subtitle or "")
+	elseif not IsValid(subtitle.color) then
```

Comment:
same as above

Reply:
this is always valid. Or do you refer to the iscolor?

Reply:
Yep, same as above :D